### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by cloudhead**

### CODE REVIEW

File: radicle-node/src/wire/protocol.rs [hunk 1]
```diff
@@ -568,21 +568,37 @@ where
...
+                            io::ErrorKind::ConnectionReset,
+                        )));
+
+                        if let Some(id) = peer.id() {
+                            self.service.disconnected(*id, &reason);
+                        }
+                        peer.disconnected(reason);
+                    }
+                    None => {
+                        log::warn!(target: "wire", "Peer with fd {fd} is unknown");
```
Comment (slack-coder) [inline radicle-node/src/wire/protocol.rs:599]: Hmm is fd the file descriptor?  Has it been useful with diagnosing a node?  I haven't seen the file descriptor used in logs in other services.

File: radicle-cli/src/commands/patch/create.rs [hunk 2]
```diff
@@ -34,19 +34,19 @@ pub fn handle_patch_message(
         .message()
         .ok_or(anyhow!("commit summary is not valid UTF-8; aborting"))?;
     let message = message.get(&format!("{commit_message}{PATCH_MSG}"));
+    let message = message.replace(PATCH_MSG.trim(), ""); // Delete help message.
```
Comment (slack-coder) [inline radicle-cli/src/commands/patch/create.rs:37]: Nice, though we should eventually improve `message.get()` to handle this

**Comment by slack-coder** [comment]
otherwise :+1:

File: radicle-node/src/wire/protocol.rs (hunk 1 above)
Comment (cloudhead) [reply radicle-node/src/wire/protocol.rs:599]: Yeah. We use it everywhere in the wire protocol logs, as it's the unique identifier for a given peer.

File: radicle-cli/src/commands/patch/create.rs (hunk 2 above)
Comment (cloudhead) [reply radicle-cli/src/commands/patch/create.rs:37]: Agreed.

File: radicle-node/src/wire/protocol.rs [hunk 3]
```diff
@@ -568,21 +568,37 @@ where
                 // TODO: This should be a fatal error, there's nothing we can do here.
                 log::error!(target: "wire", "Can't poll connections: {}", err);
             }
-            reactor::Error::ListenerPollError(id, err) => {
+            reactor::Error::ListenerPollError(id, _) => {
                 // TODO: This should be a fatal error, there's nothing we can do here.
-                log::error!(target: "wire", "Received error: listener {} disconnected: {}", id, err);
+                log::error!(target: "wire", "Received error: listener {} disconnected", id);
```
Comment (FintanH) [inline radicle-node/src/wire/protocol.rs:573]: Were the errors too much noise or not useful?

Comment (cloudhead) [reply radicle-node/src/wire/protocol.rs:573]: They actually weren't errors, was a bad variable name, they are the event state of the file descriptor, which is kind of useless.

**Pull request closed without merging**